src/cse-machine/interpreter.ts

```diff
@@ -311,8 +310,9 @@ export function* generateCSEMachineStateStream(
 
   let command = control.peek()
 
-  // First node will be a Program
```

Comment:
Is this deleted comment still relevant?

Reply:
I'll replace it with a comment that more accurately describes the new code.